docs/developer-docs/latest/developer-resources/global-strapi/api-reference.md

```diff
@@ -60,7 +60,7 @@ Returns a function that parses the configurations, hooks, middlewares and APIs o
 
 ## strapi.log
 
-Returns the Logger ([Winston](https://github.com/winstonjs/winston)) instance.
+Returns the Logger (Pino) instance.
```

Comment:
:confused: I thought we moved from Pino To Winston, not the reverse?

Reply:
Definitely! Good catch!

Probably something went wrong when I fixed some conflicts (also, this should not be in this PR but in the ["lighter updates"](https://github.com/strapi/documentation/pull/430) one 🤷 )

Reply:
```suggestion
Returns the Logger ([Winston](https://github.com/winstonjs/winston)) instance.
```